sopel/tools/jobs.py

```diff
@@ -20,20 +20,14 @@
     import queue as Queue
 
 
-class released(object):
-    """A context manager that releases a lock temporarily."""
-    def __init__(self, lock):
-        self.lock = lock
-
-    def __enter__(self):
-        self.lock.release()
-
-    def __exit__(self, _type, _value, _traceback):
-        self.lock.acquire()
+class PriorityQueue(Queue.PriorityQueue):
```

Comment:
Per the same 5.3.0 changelog entry, we shouldn't consider this deprecated, as everything in this module is already considered internal since 6.0. Just removing it should be fine.

Reply:
Good point, thank you!